igor-web/src/main/java/com/netflix/spinnaker/igor/concourse/client/model/Plan.java

```diff
@@ -45,13 +45,21 @@
     @Nullable
     private ResourceShape put;
 
+    @Nullable
+    private OnSuccess onSuccess;
+
     @Nullable
     public Resource getResource() {
-      ResourceShape shape = get == null ? put : get;
+      String resourceId = id;
+      ResourceShape shape = get;
+      if(shape == null && onSuccess != null) {
```

Comment:
(I hate only calling out minor stylistic bits) one teenie tiny space before `(`

Reply:
No worries. Appreciate you pointing it out.